frontend/src/components/OrganizationProfile.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Card from '../layouts/CardLayout';
+import Button from './Button';
+import CardHeader from './CardHeader';
+import CardBody from './CardBody';
+import FormInput from './FormInput';
+import { PencilIcon } from '@heroicons/react/24/outline';
+import FormTextarea from './FormTextarea';
+import { useTranslation } from 'react-i18next';
+
+interface IOrganization {
+  id: string;
+  name: string;
+  email: string;
+  phone: string;
+  address: string;
+  logo: string;
+  description: string;
+}
+
+interface OrganizationProfileProps {
+  organization: IOrganization;
+}
+
+const OrganizationProfile = ({ organization }: OrganizationProfileProps) => {
+  const { t } = useTranslation(['organization', 'general']);
+
+  return (
+    <Card>
+      <CardHeader>
+        <h2>{t('title')}</h2>
```

Comment:
Please validate with @OlteanuCosmin and @AndreeaMariaP for the font behaviour on mobile devices.